src/Components/ADT3DSceneBuilder/Internal/Behaviors/Behaviors.tsx

```diff
@@ -184,66 +184,79 @@ const SceneBehaviors: React.FC<Props> = ({
     const customStyles = getStyles(theme);
     return (
         <div className="cb-scene-builder-pivot-contents">
-            <SearchHeader
```

Comment:
when the scrolling shows up, the right padding feels wrong. I now see what you were talking about before where the scrollbar should be indented instead. @darsney 
![image](https://user-images.githubusercontent.com/57726991/172722594-83e3cab3-6b26-4f6c-a5fe-392b3d10ebc3.png)

Reply:
not sure how that same logic works here though. Cause this kinda feels off too. I think they would both be better if they had the padding outside the scroll. Then the scroll is under the search box.
![image](https://user-images.githubusercontent.com/57726991/172722782-ac42a685-2755-49dc-beab-bfaecb060cb1.png)

Reply:
But as a counter, if we followed the same pattern here, we would have no padding right of the text boxes??? hmmm that doesn't seem great.
![image](https://user-images.githubusercontent.com/57726991/172723016-4bd5b881-4125-4251-95e3-93b6df5ae3d0.png)

Reply:
Yeah IMO, it's better to snap the scroll bar to the far right & pad the left of the scroll bar.  It looks v jank when the content touches the scroll bar.  I think having some space between the list item hover color & the scroll bar is the best option here.